### ISSUE TITLE
track available

### ISSUE THREAD

**Issue by skypce** [jrnewell/spotify-ripper#46]
Some playlist when i download show me this : 
Loading track...
Track is not available, skipping...

If i check the song with spotify it is available.  Congratulations by this exceptional software . and thank you in advance

**Comment by jrnewell** [comment]
Which track are you trying to rip?  I think I have noticed this too, so I will try doing some research to see if there is a way around this.  Unfortunately, I am limited to what libspotify tells me is available.

**Comment by skypce** [comment]
Hello again, by example this: spotify:track:1ClQpnPU9M6mQpXGvhGPvQ

No way for download.

**Comment by skypce** [comment]
May be this info can be util:
I am  Chilean , but i dont have chilean credit card. i was used openvpn and create a account from UK .
with premium account created i was downloaded the spotify.key .I was use this key with my chilean normal local account and not the uk premium account. Can be this the problem? . When i browse between the two accounts the results are different.

**Comment by yanwhite** [comment]
Same issue here, with this track: spotify:track:5dZu9qegz7rPiSWcMpygT6
I logged in with the same account to download the key, and the track plays in spotify

**Comment by frontenddevguy** [comment]
Same issue here, with this track: spotify:track:5dZu9qegz7rPiSWcMpygT6
I logged in with the same account to download the key, and the track plays in spotify

**Comment by jrnewell** [comment]
@skypce yeah it could be licensing/regional issues because I can play the track fine (both on Spotify and spotify-ripper) - I am in the US.

@frontenddevguy What is strange is the track you gave me seems be for the regular album ``Interloper``, but if you try to find that track on Spotify, it directs you towards the "2015 Remaster" version.  That uri seems to work fine?  spotify:track:1nbxaFALcmHQ1SrbcWl934
